portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,39 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+
+<head>
```

Comment:
not sure if this is just github's UI displaying this poorly, but make sure all your tags are properly indented relative to the base `html` tag:
```
<html>
  <head>
    ...
  </head>
  <body>
    ...
  </body>
</html>
```

Reply:
Will be fixed in #7